breathecode/utils/decorators/capable_of.py

```diff
@@ -76,4 +77,11 @@ def get_academy_from_capability(kwargs, request, capability):
             f"You (user: {request.user.id}) don't have this capability: {capability} for academy {academy_id}"
         )
 
+    if request.get_full_path() != '/v1/activity/academy/cohort/1?limit=5':
```

Comment:
test `@capable_of` this is a example how test a decorator `breathecode/utils/tests/decorators/tests_has_permission.py`